src/app/components/store/canvas.state.ts

```diff
@@ -0,0 +1,5 @@
+import { fabric } from 'fabric';
+export const initialState = {
+  eventState: JSON.stringify(fabric.Canvas),
+  eventType: '',
+};
```

Comment:
It would be better if you give an interface for state. Also in the initiaState canvasState would be a more appropriate name since it holds data  contained within the canvas, you could initialize it as an empty string.